Velaptor/Input/Mouse.cs

```diff
@@ -15,13 +15,15 @@ public class Mouse : IMouseInput<MouseButton, MouseState>
         private static int xPos;
         private static int yPos;
         private static int scrollWheelValue;
+        private static MouseScrollDirection mouseScrollDirection;
 
         /// <summary>
         /// Gets the current state of the mouse.
         /// </summary>
         /// <returns>The state of the mouse.</returns>
         public MouseState GetState()
         {
+            // TODO: Check to see if we need to check '<='.  Instead use '<'
```

Comment:
Remove comment.

No need to do what the TODO comment instructs.  Mouse and keyboard functionality are going to be reworked in a future release